File: 02_junit4_java/src/test/java/jug/nsk/try_kotlin1/LibraryTest.java
package jug.nsk.try_kotlin1;

import org.junit.AfterClass;
import org.junit.BeforeClass;
import org.junit.Test;
import static org.junit.Assert.*;

// java, JUnit4
public class LibraryTest {

    @BeforeClass
    public static void setUp() {
        System.out.println("setUp");
    }

    @Test public void testSomeLibraryMethod() {
        // given
        Library classUnderTest = new Library();
        // when
        boolean result = classUnderTest.someLibraryMethod();
        // then
        assertTrue("just 'true'", result);
    }

    @AfterClass
    public static void tearDown() {
        System.out.println("tearDown");
    }

}

File: 06_junit4_on_junit5_platform_per_class/src/test/kotlin/jug/nsk/try_kotlin1/LibraryTest.kt
package jug.nsk.try_kotlin1

import org.junit.AfterClass
import org.junit.Assert.assertTrue
import org.junit.BeforeClass

class LibraryTest {

//    @Test
    fun testSomeLibraryMethod() {
        val classUnderTest = Library()
        assertTrue("just 'true'", classUnderTest.someLibraryMethod())
    }

    @BeforeClass
    fun setUp() {
        println("setUp")
    }

    @AfterClass
    fun tearDown() {
        println("tearDown")
    }

}
